components/bosch/BMX160.stanza

```diff
@@ -0,0 +1,100 @@
+#use-added-syntax(jitx)
+defpackage ocdb/bosch/BMX160 :
+  import core
+  import collections
+  import math
+  import jitx
+  import jitx/commands
+  import ocdb/defaults
+
+  import ocdb/land-patterns
+  import ocdb/symbols
+  import ocdb/box-symbol
+  import ocdb/bundles
+  import ocdb/generator-utils
+  import ocdb/generic-components
+
+public pcb-component component :
+  manufacturer = "Bosch"
+  mpn = "BMX160"
+  description = "Small, low-power, 9-axis IMU (accel, gyro, magnetometer)"
+  pin SDx
+  pin SCx
+  pin SDO
+  pin CSB
+  pin-properties :
+    [pin:Ref | pads:Int ... | side:Dir ]
+    [SDO    | 1     | Right ]
+    [GND    | 2,3,7 | Down  ]
+    [INT1   | 4     | Left  ]
+    [VDDIO  | 5     | Left  ]
+    [GNDIO  | 6     | Down  ]
+    [VDD    | 8     | Left  ]
+    [INT2   | 9     | Left  ]
+    [NC0    | 10    | Down  ]
+    [NC1    | 11    | Down  ]
+    [CSB    | 12    | Right ]
+    [SCx    | 13    | Right ]
+    [SDx    | 14    | Right ]
+
+  make-box-symbol()
+  assign-landpattern(BMX160-pkg)
+
+  supports i2c :
+    i2c.scl => SCx
+    i2c.sda => SDx
+
+  supports spi :
+    spi.sck => SCx
+    spi.ss => CSB
+    spi.mosi => SDx
+    spi.miso => SDO
+
+pcb-landpattern BMX160-pkg :
+  val dx = 0.675 / 2.0 + 0.925
+  val dy = 0.675 / 2.0 + 0.675
+  val pitch = 0.5
+  pad p[1] : smd-pad(0.675, 0.25) at loc((- dx), (1.5 * pitch))
+  pad p[2] : smd-pad(0.675, 0.25) at loc((- dx), (0.5 * pitch))
+  pad p[3] : smd-pad(0.675, 0.25) at loc((- dx), ((-0.5) * pitch))
+  pad p[4] : smd-pad(0.675, 0.25) at loc((- dx), ((-1.5) * pitch))
+
+  pad p[7] : smd-pad(0.25, 0.675) at loc(pitch, (- dy))
+  pad p[6] : smd-pad(0.25, 0.675) at loc(0.0, (- dy))
+  pad p[5] : smd-pad(0.25, 0.675) at loc((- pitch), (- dy))
```

Comment:
why reverse order on the pad numbering here?

Reply:
Looks like this was done manually, which means that the engineer is following the pattern in the mechanical drawing in the datasheet - so not necessarily in numeric order.